content/en/blog/2024/announcing-notation-v1-2.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Notary Project announces Specification v1.1.0 and Notation v1.2.0!
+author:  "Notary Project Release Team"
+date: 2024-08-30
+draft: false
+---
+
+The Notary Project maintainers are excited to announce new releases, including [Notary Project specifications v1.1.0](https://github.com/notaryproject/specifications/releases/tag/v1.1.0), [notation v1.2.0](https://github.com/notaryproject/notation/releases/tag/v1.2.0), [notation-go v1.2.0](https://github.com/notaryproject/notation-go/releases/tag/v1.1.0), and [notation-core-go v1.1.0](https://github.com/notaryproject/notation-core-go/releases/tag/v1.1.0). These versions are now ready for production use!
+
+## Deprecation
+
+The experimental flag `--allow-referrers-api` used by `notation sign` and `notation verify` commands is now deprecated. See [Support OCI specification v1.1.0](#support-oci-specification-v110) for details.
+
+## Notable Capabilities in this Release
+
+Here are some of the major capabilities and features included in this release.
+
+### Notary Project specifications
+
+The Notary Project specifications now include support for [RFC 3161](https://www.rfc-editor.org/rfc/rfc3161) timestamping and introduce Notation plugin conventions in the [plugin specification](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/plugin-extensibility.md).
+
+### Support OCI specification v1.1.0
+
+In Feb 2024, the Open Container Initiative (OCI) community released version 1.1.0, which includes the [OCI image specification v1.1.0](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0) and the [OCI distribution specification v1.1.0](https://github.com/opencontainers/distribution-spec/releases/tag/v1.1.0). Notation now adheres to the OCI spec v1.1.0, leading to the deprecation of the experimental flag `--allow-referrers-api`. A new flag, `--force-referrers-tag` (default to `true`), has been introduced to the `notation sign` command, enabling users to choose the referrers tag schema over the [referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0/spec.md#enabling-the-referrers-api) if the registry they are using does not yet support the referrers API. The `notation verify/list/inspect` commands attempt to use the referrers API first and automatically fall back to the [referrers tag schema](https://github.com/opencontainers/distribution-spec/blob/v1.1.0/spec.md#referrers-tag-schema) if the referrers API is not supported by the registry.

```

Comment:
If all registries support referrers API, then there is no need for adding this flag `--force-referrers-tag`.  We want referrers API to be the default one, this is the goal. However, not all registries currently support referrers API. Another reason is we do not want to introduce breaking changes in v1.2.0, since version v1.10 and prior versions by default use tag schema. 

In Notation v2.0, we will introduce the breaking change, so that by default Notation cli will use referrers API, however, user can choose to use tag schema even if registries support referrers API by setting `--force-referrers-tag` to `true`

Reply:
Read this sentence from the blog post:

> The `notation verify/list/inspect` commands attempt to use the referrers API first and automatically fall back to the [referrers tag schema](https://github.com/opencontainers/distribution-spec/blob/v1.1.0/spec.md#referrers-tag-schema) if the referrers API is not supported by the registry.

This is the proper OCI 1.1 client behavior, i.e., the client must check to see if the registry supports the referrers API before trying to use it and if the registry does not support the referrers API the client must use the referrers tag fallback.

You say:

> If all registries support referrers API, then there is no need for adding this flag `--force-referrers-tag`.

If notation is OCI 1.1 compliant, it does not matter if all registries support the referrers API or not. The _only_ use of the `--force-referrers-tag` command-line option is if the client wants to use the referrers tag fallback _even if a registry supports the referrers API._ If a registry does not support the referrers API, the client _has_ to use the referrers tag fallback, there is no other option. You acknowledge this in this later statement:

> user can choose to use tag schema even if registries support referrers API by setting `--force-referrers-tag` to `true`

Therefore, this statement in the blog is misleading/wrong:

> A new flag, `--force-referrers-tag` (default to `true`), has been introduced to the `notation sign` command, enabling users to choose the referrers tag schema over the [referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0/spec.md#enabling-the-referrers-api) if the registry they are using does not yet support the referrers API.

The end of the sentence should read "… even if the registry they are using supports the referrers API."

Reply:
@whalelines I think I understood your point now. You mean with the current default behavior, tag schema is used over the referrers API even if the registry they are using supports the referrers API. I will create a PR to amend the blog. Thanks.